binderhub/static/index.js

```diff
@@ -258,7 +257,6 @@ $(function(){
         image.onStateChange('waiting', function(oldState, newState, data) {
             $('#phase-waiting').removeClass('hidden');
         });
-
         image.onStateChange('building', function(oldState, newState, data) {
             $('#phase-building').removeClass('hidden');
```

Comment:
If I add a update_fav here, seem to work for me. We can even just put it just after `$('#build-form').submit(function() {` (about line 230)

Reply:
ok - updated moving it to the section you suggested